src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -586,6 +586,7 @@ def subvolume_snapshot_info(self, **kwargs):
             with open_volume(self, volname) as fs_handle:
                 with open_group(fs_handle, self.volspec, groupname) as group:
                     with open_subvol(self.mgr, fs_handle, self.volspec, group, subvolname, SubvolumeOpType.SNAP_INFO) as subvolume:
+                        subvolume.check_dangling_clone_index(snapname)
```

Comment:
this check is part of `subvolume_snapshot_info()`? so that means that in order to get rid of the non-existent pending clones we need to use snapshot info cmd? Won't this look odd?

Reply:
Followed this: https://tracker.ceph.com/issues/58090#note-8

Reply:
I had a similar concern reviewing the PR, but the tracker comment explained it. It would be great if the PR and the commit both had a summary that explained it better:

```
mgr/volumes: cleanup dangling clone symlinks on snapshot info

Older versions may have produced dangling clone symlinks
which remained post-upgrad. We are now cleaning those up
in the snapshot info command.
```